schema/deploy/tables/application_revision.sql

```diff
@@ -8,6 +8,7 @@ create table ggircs_portal.application_revision (
     version_number int not null,
     certification_signature bytea,
     legal_disclaimer_accepted boolean not null default false,
+    -- TODO: This should be renamed. It is vague. The user_id here is the id of the certifier, not the creator of the row
```

Comment:
If you make a card for this, it would be a good opportunity to explore graphile field deprecation and aliasing as a team.